src/user/components/config.js

```diff
@@ -0,0 +1,115 @@
+const config = {
```

Comment:
issue: add copyright notice

Reply:
suggestion: rename this file. we already have `src/config.js`. Perhaps storymapconfig?

Reply:
@paulschreiber this is a test file that I forgot to delete, I will remove it